client/src/pages/dashboard/NewEventType.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Container from '../../components/Layout/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '../../components/DataDisplay/Typography';
+import Button from '../../components/Inputs/Button';
+import TextField from '@material-ui/core/TextField';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
+
+const NewEventType = () => {
+    return (
+        <Container>
+            <Grid container justify="space-between" alignItems="center" spacing={2} >
+                <Grid item >
+                    <Button
+                        component={Link} 
+                        to="/event_types"
+                        variant="outlined"
+                        color="primary"
+                        text="Back"
+                    />
+                </Grid> 
+               <Grid item>
+                <Typography 
+                    variant="subtitle1" 
+                    component="h1"
+                    text="Add One-on-One Event Type"
+                    />
+               </Grid>
+               <Grid item>
+                    <Typography 
+                        variant="subtitle2" 
+                        component="p"
+                        text="Your event type is OFF"
+                    />
+               </Grid>
+            </Grid>
+            <form>
+                <Grid container spacing={2}>
+                    <Grid item xs={12}>
+                        <TextField
+                            autoComplete="name"
+                            name="name"
+                            variant="outlined"
+                            required
+                            fullWidth
+                            id="name"
+                            label="Event Name"
+                            autoFocus
+                        />
+                    </Grid>
+                    <Grid item xs={12}>
```

Comment:
```suggestion
<Grid item container>
                    <Grid item xs={12} md={6}>
</Grid>
```

this would allow the mobile to span full width and desktop/tablet will only take half of the width